### ISSUE TITLE
Building waypoints cannot be placed on buildings that are not visible in Fog

### ISSUE THREAD

**Issue by xezon** [xezon/GeneralsGamePatch#197]
Building waypoints cannot be placed on buildings that are not visible in Fog.

**Comment by Jundiyy** [comment]
Hmmm, would remove way point scouting.

**Comment by commy2** [comment]
^ This. I would call this a "big change".

**Comment by Josef-Vonasek** [comment]
I don't know. I have seen it used only in 2% of pro matches. Not to mention, it's a pretty apparent bug exploit.

**Comment by RisingZH** [comment]
Whilst I would call this a bug exploit I do think waypoint scouting is beneficial for the game and would therefore not support removing it.

**Comment by xezon** [comment]
Code issue. We cannot touch this in Patch.

**Comment by xezon** [comment]
Let's reopen it for now to invite discussion. Could be fixed in Thyme for sure, however, we currently do not plan for Thyme, so any discussions here cannot be taken into consideration for any potentially related design decisions.

**Comment by RisingZH** [comment]
Bumping this one as a couple people have brought it up saying it should be removed on Gamereplays.

I think I’ve changed my mind compared to my previous post above. Whilst I think waypoint scouting is beneficial on 1.04 as it helps scout things like no eco, if the game was better balanced in the first place it would be unnecessary.

It definitely is a bug and you wouldn’t find it in any other rts games. Scouting is an important part of any rts and you shouldn’t be able to do it just by being able to click a building into the fog.

**Comment by xezon** [comment]
I agree with the statement that it can't be treated for anything else than it is, a bug.
